package.json

```diff
@@ -60,6 +63,13 @@
     "@types/storybook__react": "^4.0.0",
     "@typescript-eslint/eslint-plugin": "2.2.0",
     "@typescript-eslint/parser": "2.2.0",
+    "@wdio/cli": "5.11.14",
+    "@wdio/jasmine-framework": "5.12.1",
+    "@wdio/local-runner": "5.12.1",
+    "@wdio/sauce-service": "5.12.1",
+    "@wdio/selenium-standalone-service": "5.12.1",
+    "@wdio/spec-reporter": "5.12.1",
+    "@wdio/static-server-service": "5.12.1",
```

Comment:
I think I asked this question already but I'm no longer sure what's the answer. 
Is it not possible to hide this away in `instantsearch-e2e-tests` ?

Reply:
I also think we should hide this

Reply:
It would also make sense to me.

Reply:
Agree.

Reply:
Well... I'm not for this.

First of all there is very few advantages (if any?) to "hide" dependencies but the real problem is mainly that it will makes us call a subdependency from the parent project (`react-instantsearch` calls `wdio` directly), which is not recommenced (the package version we're targeting may not be at the `node_modules` top level, depending how npm/yarn deduplicated the dependencies).

Reply:
can we also hide `wdio` call then?

Reply:
What would be the objective of hiding these dependencies?

Reply:
It would remove the overall perceived complexity, as well as reduce the noise and cognitive overhead of managing these dependencies.

I find it surprising that we went all the way to packaging the E2E tests (e.g. see also `wdio.saucelabs.conf.js` which only exports the E2E package import), but are reluctant to hide the dependencies themself.

The E2E test suite would be more predictable if the `wdio` dependencies are packaged within it. Here, we are responsible for the `wdio` deps to work with a black-boxed test suite, which is quite contradictory.